application-xpoll-ui/src/main/resources/XPoll/XPollJSExtension.xml

```diff
@@ -151,6 +151,23 @@
         }
       });
     });
+    $('.xpollSelect').change(function() {
+      var changedElement = $(this);
+      $('.xpollSelect').each(function() {
+        if ($(this).data('proposal') != changedElement.data('proposal') &amp;&amp; $(this).val() ==
+        changedElement.val())
+          $(this).find('option:eq(0)').prop('selected', true);
```

Comment:
```suggestion
          // Reset the value because we can't have two proposals with the same value (index).
          $(this).val('');
```

Reply:
If i do this, the select will become blank, rather than "Select a value" (or the translated version of it).
![image](https://user-images.githubusercontent.com/45433221/135275570-e48970f1-3b2a-4368-baee-663e90c758e8.png)

Reply:
That's because the label "Select a value" is not associated with the empty value I think. Can you try putting ``value=""`` on that default option?

Reply:
I did and it works 💪